src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Data
     /// A column to hold primitive types such as int, float etc.
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    public partial class PrimitiveColumn<T> : BaseColumn, IEnumerable<T?>
```

Comment:
You need to still keep `IEnumerable<T?>`. This way things like XPlot charting still work by just passing it a Column, and it enumerates over the data in the column.